x-pack/plugin/fleet/src/main/java/org/elasticsearch/xpack/fleet/Fleet.java

```diff
@@ -73,7 +73,7 @@
  */
 public class Fleet extends Plugin implements SystemIndexPlugin {
 
-    private static final int CURRENT_INDEX_VERSION = 7;
+    private static final int CURRENT_INDEX_VERSION = 8;
```

Comment:
Changing this value is causing test failures, because we have the `-7`suffix hardcoded in some of them. The current index version is used in the index name, and I think we want to wait to bump it to 8 until we are reindexing for the 8.x release.

Since this mapping change is only adding a value and not changing a type, the `SystemIndexManager` class should cleanly apply the change. It will use the Elasticsearch version number under the hood to determine whether the mappings are out of date, and won't need to change the index name.

In short, I think we can back out the change in this line and get tests to pass.

Reply:
Thank you for the feedback. Rolled back the version bump.